crates/net/network/src/cache.rs

```diff
@@ -9,12 +19,19 @@ use std::{borrow::Borrow, hash::Hash, num::NonZeroUsize};
 pub struct LruCache<T: Hash + Eq> {
     limit: NonZeroUsize,
     inner: LinkedHashSet<T>,
+    tx: Option<UnboundedSender<T>>,
```

Comment:
This is cool, but I would add some docs for this and change the name from `tx` to something that makes it more clear that this is the eviction feedback sender

Reply:
also some docs for the `LruCache` struct about this behavior

Reply:
valid, will make a new type though https://github.com/paradigmxyz/reth/pull/6016#discussion_r1448456693